app/src/androidTest/java/org/mozilla/focus/helpers/TestAssetHelper.kt

```diff
@@ -29,7 +29,7 @@ object TestAssetHelper {
     }
 
     fun getPlainPageAsset(server: MockWebServer): TestAsset {
-        val url = server.url("plain_test.html").toString()
+        val url = server.url("plain text page.html").toString()
```

Comment:
I think we should rename the page similar to what we use in Fenix, e.g, `generic`.html